index.js

```diff
@@ -123,11 +123,21 @@ bot.on('messageCreate', async message => {
         message.reply(`Essa braba foi pra fila: ${song.title} (${song.durationFormatted})`)
       }
 
+      if (IDLE_STATE) {
```

Comment:
@igormath I think it needs a review: which problem are you trying to solve here?